### PR TITLE
Web live preview

### LINKED ISSUE #809: Create Doc on why some external resources in a user's web page are blocked from showing in the Web Live Preview designer

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #809
Fixes https://github.com/dotnet/AspNetCore.Docs/issues/28471

[Internal review URL](https://review.learn.microsoft.com/en-us/aspnet/web-forms/overview/getting-started/?branch=pr-en-us-812#why-are-external-resources-in-my-page-being-blocked-in-web-live-preview)

### CODE REVIEW

File: aspnet/web-forms/overview/getting-started/index.md [hunk 1]
```diff
@@ -33,3 +33,5 @@ msc.type: chapter
     - [What's New in ASP.NET and Web Development in Visual Studio 2012](hands-on-labs/whats-new-in-aspnet-and-web-development-in-visual-studio-2012.md)
     - [What's New in Web Forms in ASP.NET 4.5](hands-on-labs/whats-new-in-web-forms-in-aspnet-45.md)
     - [Using Page Inspector in Visual Studio 2012](hands-on-labs/using-page-inspector-in-visual-studio-2012.md)
+
+[!INCLUDE[](~/includes/webLivePreview.md)]
```
Comment (Rick-Anderson) [inline aspnet/web-forms/overview/getting-started/index.md:37]: @bzhu94 copy/paste this line to have the content added to any file.

Comment (Rick-Anderson) [reply aspnet/web-forms/overview/getting-started/index.md:37]: To add this to any article, At the top right of the article, select the Edit pen icon:

image

Select the pen icon again.  Edit, then Save.

File: aspnet/includes/webLivePreview.md [hunk 2]
```diff
@@ -1,53 +1,55 @@
...
+* Synchronizing the contents and currently selected node in the designer with that of the editor.
+* Pushing new content created from the designer into the editor.
+* Executing Action Panel commands.
 
-## Potential Security Vulnerabilities with Web Live Preview / BrowserLink:
+## Potential Security Vulnerabilities with Web Live Preview and BrowserLink
 
-If you inject any scripts or resources from a 3rd party into your web page, this could expose you to a potential security vulnerability - leveraging WLP / BrowserLink to make a Cross-Site Scripting (XSS) attack.
+If you inject any scripts or resources from a third party into your web page, this could expose you to a potential security vulnerability by using WLP and BrowserLink to make a [Cross-Site Scripting (XSS) attack](/aspnet/web-forms/videos/how-do-i/how-do-i-understand-and-defend-against-script-injection-attacks-in-aspnet).
```
Comment (Rick-Anderson) [inline aspnet/includes/webLivePreview.md:19]: @bzhu94 replace `-` with `by` as `-` wont MT (Machine Translate).

**Review by bzhu94 — APPROVED** [review]
Looks good to me!

**Pull request merged**